src/ol/interaction/ModifyPolygonBrush.js

```diff
@@ -103,6 +110,25 @@ class ModifyPolygonBrush extends Modify {
     }
   }
 
+  updateSketchRadiusByPressure_(event) {
+    if (this.sketchPoint_ && event.pointerEvent.pressure != 0) {
+      this.sketchPointRadius_ = getNewSketchPointRadius(event, this.sketchPointRadius_);
+      this.sketchRadius_ = Math.max(
```

Comment:
You could add a new `getNewSketchPointRadiusByPressure` function to the `PolygonBrush` file and import it here just as `getNewSketchPointRadius`. This way you don't have to duplicate code (e.g. this function and `MIN_BRUSH_SIZE`).